_licenses/eupl-1.1.txt

```diff
@@ -0,0 +1,346 @@
+﻿---
+title: European Union Public License 1.1
+nickname: EUPL-1.1
+tab-slug: eupl-v1.1
+redirect_from: /licenses/eupl-v1.1/
+family: EUPL
+featured: false
+source: https://joinup.ec.europa.eu/community/eupl/og_page/european-union-public-licence-eupl-v11
+
+description: The “European Union Public Licence” (EUPL) The EUPL is the first European Free/Open Source Software (F/OSS) licence. It has been created on the initiative of the European Commission. It is now approved by the European Commission in 22 official languages of the European Union.
+
+how: Create a text file (typically named COPYING or LICENCE.txt) in the root of your source code and copy the text of the license into the file.
+
+note: The European Commission recommends taking the additional step of adding a boilerplate notice to the top of each file. The boilerplate can be found at https://joinup.ec.europa.eu/sites/default/files/ckeditor_files/files/EUPL%201_1%20Guidelines%20EN%20Joinup.pdf
+
+required:
+  - include-copyright
+  - disclose-source
+  - document-changes
+  - network-use-disclose
+
+permitted:
+  - commercial-use
+  - modifications
+  - distribution
+  - patent-use
+  - private-use
+  - sublicense combined derivatives under a compatible licence
```

Comment:
sublicense should be removed; properties don't allow for qualification here, and this one is used only for licenses that explicitly permit sublicensing, not ones that allow derivatives under a compatible license